TASK/TaskWarpPedDirectlyIntoCover.md

```diff
@@ -0,0 +1,22 @@
+---
+ns: TASK 
+---
+
+## TASK_WARP_PED_DIRECTLY_INTO_COVER
+
+```c
+// 0x6E01E9E8D89F8276 
+void TASK_WARP_PED_DIRECTLY_INTO_COVER(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5);
+```
+
+```
+NativeDB Introduced: v2545
+```
+
+## Parameters
+* **p0**:
+* **p1**:
+* **p2**:
+* **p3**:
+* **p4**:
+* **p5**:
```

Comment:
* **ped**: The ped index.
* **canpeek**: If ped can peek and aim. False by default.
* **forceinitialdirection**: False by default.
* **forcefaceleft**: False by default. 
* **coverpoint**: Cover index.


I don't seem any `p5`.